Week2/Assignment3/src/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Namaste React Bootcamp | Assignment 3</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div id="root">
+      <!-- Some error occurred -->
+    </div>
+    <script type="module" src="App.js"></script>
```

Comment:
Can put this in the head tag. Modules are deferred by default.

Reply:
Interesting I didn't know this